docs/_scss/components/_accordions.scss

```diff
@@ -11,21 +11,6 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-top: 1rem;
```

Comment:
Thanks @msecret! 

I thought that's where we were going with gulp.

Would the `gulpfile.js` found here help inform the fix you are looking to make with the gulp build? (https://github.com/BrianGilmanGSA/sam-web-design-standards)

Right now I *cannot* run a gulp build (see #1148) - I can only do `$ ./go serve` to confirm the changes, which doesn't update the `/docs`. If you could help me out on that score, I would be more than happy to make changes and resubmit along with using the `/src` directory for future PRs.

Thanks again.